vault/project.n.2020.qol.md

```diff
@@ -5,6 +5,9 @@ desc: ''
 updated: 1599501500764
 created: 1599501500764
 stub: false
+start: 2020.10.07
+end: 2020.10.29
+status: In Progress
```

Comment:
can you add related tasks from the backlog to this project that make sense? this might also be good to do a community poll on

Reply:
Yes, but those can be separate commits yeah? As we take suggestions from the community :)